library/core/lifecycle_events/src/main/java/org/quiltmc/qsl/lifecycle/api/client/event/ClientLifecycleEvents.java

```diff
@@ -52,15 +52,15 @@
 	 * was running an integrated server, the integrated server will be shut down. Finally, all client facilities are torn down.
 	 *
 	 * <h2>What should mods do when this event is executed?</h2>
-	 *
+	 * <p>
```

Comment:
Do we have a convention on where to put <p> tags in javadoc? In most of the existing code it seems to be on the first line of a paragraph, in this PR it seems to always be on its own line.

Reply:
Putting those in the first line of a paragraph was always unreadable to me, it's an old checkstyle thing on Fabric.

I think they should be on their own line in Quilt.